core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -6295,8 +6290,7 @@ protected function formatGroupHeader($field, $value, $mode, $row)
 	 */
 	protected function initRoots()
 	{
-		$table = $this->strTable;
```

Comment:
removed `$table` here (and it's usage in the `elseif`below), because it is re-defined in the `if ($this->treeMode)` condition below. Technically would work, but I found the code confusing like this.